neo/app/controllers/Application.scala

```diff
@@ -47,6 +49,40 @@ class Application @Inject()(val wsClient: WSClient,
   val listener = system.actorOf(Props(classOf[Listener], this))
   system.eventStream.subscribe(listener, classOf[DeadLetter])
 
+  //start loading city data
+  val file = environment.getFile("/public/data/city.sample.json")
+  val stream = new FileInputStream(file)
+  val json = Json.parse(stream)
+  val values = (json \ "features").as[List[JsValue]]
+  for(v <- values) {
+    val tp = (v \ "geometry" \ "type").as[String]
+    var sumX = 0.0
+    var sumY = 0.0
+    var length = 0.0
```

Comment:
length is for calculating the centriod. the x coordinate of the centriod is sumX / length, and y coordinate is sumY/length. I haven't decide how to store the results yet, so I didn't do the division in the code.